app/config.py

```diff
@@ -450,6 +450,7 @@ class Config(object):
     BROADCAST_ORGANISATION_ID = "38e4bf69-93b0-445d-acee-53ea53fe02df"
 
     DVLA_API_ENABLED = os.environ.get("DVLA_API_ENABLED") == "1"
+    DVLA_API_POSTAGE_TYPE_EXCLUDE_LIST = json.loads(os.environ.get("DVLA_API_POSTAGE_TYPE_EXCLUDE_LIST", "[]"))
```

Comment:
As long as we're super careful about making sure all the env vars are in place this is fine, but a more cautious approach might be to either have `DVLA_API_POSTAGE_TYPE_INCLUDE_LIST` or to specify all of the postage types as the default if the env var isn't set?